Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -1283,7 +1283,7 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
     double viewScaleFactor() const { return m_viewScaleFactor; }
     void scaleView(double scale);
     void setShouldScaleViewToFitDocument(bool);
-    
+
```

Comment:
I guess this new line is added unintentionally, we could narrow this patch's scope into WC and Win if we remove this line.

Reply:
yeah, I will delete it, thank-you.